Dockerfile.openshift-appliance

```diff
@@ -16,7 +16,7 @@ VOLUME $ASSETS_DIR
 ENV ASSETS_DIR=$ASSETS_DIR
 
 # Install and config libguestfs
-RUN dnf -y install libguestfs-tools coreos-installer && dnf clean all
+RUN dnf -y install libguestfs-tools coreos-installer skopeo && dnf clean all
```

Comment:
also podman seems to be missing

Reply:
will be included in https://github.com/openshift/appliance/pull/18